File: DemoApp/TblModel/TblMWUserPermission.java
package com.yrkj.ar.bean;
import com.yrkj.lib.db.BaseDataModule;
import com.yrkj.lib.db.DataColumnInfo;
import com.yrkj.lib.db.SqlCommonFn;
import com.yrkj.lib.db.SqlCommonFn.DataColumnType;

public class TblMWUserPermission extends BaseDataModule {
    private static String TableName = "MWUserPermission";
    public TblMWUserPermission(){
    }
    public static String getFormatTableName(){
        return SqlCommonFn.FormatSqlTableNameString(TableName);
    }

    public static DataColumnInfo[] Columns = 
            new DataColumnInfo[]{
        new DataColumnInfo(true,false,false,false,"id",DataColumnType.INT,10),
        new DataColumnInfo(false,true,false,false,"UserGroupId",DataColumnType.INT,10),
        new DataColumnInfo(false,true,false,false,"FuncGroupId",DataColumnType.INT,10)
    };

    public static DataColumnInfo getIdColumn(){
        return Columns[0];
    }
    public static DataColumnInfo getUserGroupIdColumn(){
        return Columns[1];
    }
    public static DataColumnInfo getFuncGroupIdColumn(){
        return Columns[2];
    }

    public int id = 0;
    public int UserGroupId = 0;
    public int FuncGroupId = 0;

    public int getId() {
        return  id;
    }
    public void setId(int id) {
        this.id = id;
    }
    public int getUserGroupId() {
        return  UserGroupId;
    }
    public void setUserGroupId(int UserGroupId) {
        this.UserGroupId = UserGroupId;
    }
    public int getFuncGroupId() {
        return  FuncGroupId;
    }
    public void setFuncGroupId(int FuncGroupId) {
        this.FuncGroupId = FuncGroupId;
    }



}

File: MWR/Source/MWRSolution/MWRDBModule/DBModule/TblMWUserPermission.cs
using System;
using System.Collections.Generic;
using System.Text;
using ComLib.db;

namespace YRKJ.MWR
{

    public class TblMWUserPermission : BaseDataModule
    {

        private static string TableName = "MWUserPermission";
        public TblMWUserPermission()
        {
        }
        public static string getFormatTableName()
        {
            return SqlCommonFn.FormatSqlTableNameString(TableName);
        }

        public static DataColumnInfo[] Columns = 
                new DataColumnInfo[]{
            new DataColumnInfo(true,false,false,false,"id",SqlCommonFn.DataColumnType.INT,10),
            new DataColumnInfo(false,true,false,false,"UserGroupId",SqlCommonFn.DataColumnType.INT,10),
            new DataColumnInfo(false,true,false,false,"FuncGroupId",SqlCommonFn.DataColumnType.INT,10)
        };

        public static DataColumnInfo getIdColumn()
        {
            return Columns[0];
        }
        public static DataColumnInfo getUserGroupIdColumn()
        {
            return Columns[1];
        }
        public static DataColumnInfo getFuncGroupIdColumn()
        {
            return Columns[2];
        }

        private int _id = 0;
        private int _UserGroupId = 0;
        private int _FuncGroupId = 0;

        public int id
        {
            get
            {
                return _id;
            }
            set
            {
                _id = value;
            }
        }
        public int UserGroupId
        {
            get
            {
                return _UserGroupId;
            }
            set
            {
                _UserGroupId = value;
            }
        }
        public int FuncGroupId
        {
            get
            {
                return _FuncGroupId;
            }
            set
            {
                _FuncGroupId = value;
            }
        }

         public override void SetValue(System.Data.DataRow row)
         {
             System.Data.DataColumnCollection dataCols = row.Table.Columns;
             if(dataCols.Contains("id"))
                 SetValue(ref _id, row["id"]);
             if(dataCols.Contains("UserGroupId"))
                 SetValue(ref _UserGroupId, row["UserGroupId"]);
             if(dataCols.Contains("FuncGroupId"))
                 SetValue(ref _FuncGroupId, row["FuncGroupId"]);
         }


    }
}
